src/main/asciidoc/workingwithapachesyncope/adminconsole/topology.adoc

```diff
@@ -0,0 +1,26 @@
+//
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+===== Topology
+The topology tab provides a mapped view of the available connector bundles, instances and
+resources available and configured in the syncope deployment. It also allows the admin to perform
+all the actions listed in the realms tab.
+
+image::consoleTopology.png[console-topology]
```

Comment:
It might be better to include here the screenshot from `-Pembedded` rather than the one from standalone distribution, since in the latter there is no local connector bundles, only connector server.